charts/vineyard/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 0.18.2
+version: 0.18.2
```

Comment:
Better not relate `version` to `appVersion` according to [Helm‘s doc](https://helm.sh/docs/topics/charts/#the-appversion-field) .

Reply:
Make sense. Thanks.